packages/foreman/foreman/foreman.spec

```diff
@@ -677,6 +677,20 @@ Configuration files for the Performance Co-Pilot integration
 %{_sysconfdir}/pcp/proc/%{name}-hotproc.conf
 %{_sharedstatedir}/pcp/config/pmlogconf/%{name}-hotproc
 
+%package profiling
+Summary: Adds rails-profiling extensions to foreman
+Requires: rubygem(rack-mini-profiler)
+Requires: rubygem(memory_profiler)
+Requires: rubygem(stackprof)
```

Comment:
Please add markers so our scripts can update the content based on the bundler.d file.
```suggestion
# start specfile profiling Requires
Requires: rubygem(rack-mini-profiler)
Requires: rubygem(memory_profiler)
Requires: rubygem(stackprof)
# end specfile profiling Requires
```